ArduCopter/Log.cpp

```diff
@@ -360,7 +362,7 @@ struct PACKED log_Rate_Thread_Dt {
 // pos_target_m is lat, lon, alt OR offset from ekf origin in m
 // terrain should be 0 if pos_target_m.z is alt-above-ekf-origin, 1 if alt-above-terrain
 // vel_target_ms is m/s
-void Copter::Log_Write_Guided_Position_Target(ModeGuided::SubMode submode, const Vector3p& pos_target_m, bool is_terrain_alt, const Vector3f& vel_target_ms, const Vector3f& accel_target_mss)
+void Copter::Log_Write_Guided_Position_Target(ModeGuided::SubMode submode, const Vector3p& pos_target_m, bool is_terrain_alt, const Vector3f& vel_target_ms, const Vector3f& accel_target_mss, const float yaw_rad, const float yaw_rate_rads)
```

Comment:
```suggestion
void Copter::Log_Write_Guided_Position_Target(ModeGuided::SubMode submode, const Vector3p& pos_target_m, bool is_terrain_alt, const Vector3f& vel_target_ms, const Vector3f& accel_target_mss, const float yaw_target_rad, const float yaw_target_rate_rads)
```

Reply:
I added the above renaming changes to the code